tests/HttpClientTest.php

```diff
@@ -12,18 +12,18 @@
 // the License.
 
 use Meplato\Store2\HttpClient;
+use PHPUnit\Framework\Attributes\Group;
 
 /**
  * Tests the Meplato Store 2 HTTP client.
  */
-class HttpClientTest extends BaseTest
+class HttpClientTest extends BaseTestCase
 {
 	/**
 	 * Test if the HTTP client works, in general.
-	 *
-	 * @group root
-	 * @group http
 	 */
+	#[Group('root')]
+	#[Group('http')]
 	public function testNewHttpClient()
```

Comment:
This test fails when store is in maintenance mode.
Do not know how to feel about that.

Reply:
I think that's fine, even though it has kind of an "integration" test feel to it. While on the subject though - do we still need the store2 subdomain?

Reply:
`store2` refers to the API (`api/v2`), not the application.